x-pack/plugins/cases/public/components/visualizations/actions/add_to_existing_case.tsx

```diff
@@ -75,20 +75,25 @@ export const createAddToExistingCaseLensAction = ({
     currentAppId = appId;
   });
 
-  const owner = getCaseOwnerByAppId(currentAppId);
-  const casePermissions = canUseCases(applicationService.capabilities)(owner ? [owner] : undefined);
-
   return createAction<ActionContext>({
     id: ACTION_ID,
     type: 'actionButton',
     getIconType: () => 'casesApp',
     getDisplayName: () => ADD_TO_EXISTING_CASE_DISPLAYNAME,
-    isCompatible: async ({ embeddable }) =>
-      !isErrorEmbeddable(embeddable) &&
-      isLensEmbeddable(embeddable) &&
-      casePermissions.update &&
-      casePermissions.create &&
-      hasInput(embeddable),
+    isCompatible: async ({ embeddable }) => {
```

Comment:
Just curious why this fixed the issue?

Reply:
If it's about create case button the flyout wouldn’t appear, it's fixed here:
https://github.com/elastic/kibana/pull/154918/commits/e1e671c22282f5e6f5033f6f502631b3e5f12e8f#diff-4b6429a0bd15d6fd67fb5734dd24068c314f12abc8a1ecbe49f2006adbef73f5R291

Reply:
Do you mean about the owner moved inside `isCompatible`. If yes, it is because when `createAddToExistingCaseLensAction` is being called the `currentAppId` is not set yet. But `isCompatible` and `execute` are being called after the render of the app where the `currentAppId` is set. We need to get the `owner` after `currentAppId` is set.